src/http/AuthHttpClient.js

```diff
@@ -452,6 +452,25 @@ class AuthHttpClient {
         return this._instance(request);
     }
 
+    /**
+     * Links accounts to the member.
+     *
+     * @param {string} authorization - oauthBankAuthorization continaing bank_id and
+     * access_token
+     * @return {Object} response - response to the API call
+     */
+    async linkAccountsOauth(authorization) {
+        const req = {
+            authorization,
+        };
+        const request = {
+            method: 'post',
+            url: `/bank-accounts`,
+            data: req
+        };
+        return this._instance(request);
```

Comment:
In the case of Sberbank and N26 the response will be empty with status "FAILRUE_BANK_AUTHORIZATION_REQUIRED", meaning that a bank authorization payload (delivered by push notification in Sberbank's case) is required to complete linking. This should probably throw an exception--in the java sdk I created a BankAuthorizationRequiredException.